mayastor/src/subsys/mbus/mbus_nats.rs

```diff
@@ -18,9 +14,16 @@ pub(super) fn message_bus_init(server: String) {
     });
 }
 
+pub fn message_bus<'a>() -> &'a nats::asynk::Connection {
```

Comment:
This was originally to due with the bus trait which I've omitted for the moment to make things simpler and so the bus was to remind me that I needed to return a bus here :) 
Once we have the bus, if we have it as a trait then we can't do mbus::get() because async traits don't play well with static methods :(